### ISSUE TITLE
Making a Layer Non-Trainable

### ISSUE THREAD

**Issue by batu** [reinforceio/tensorforce#491]
Hello,

I want to use a Dense layer as a random projection, and an easy way to do it is to set it to be non-trainable. 

I read the documentation but couldn't find a way to configure to network to achieve what I want.

Is this currently supported? 

Thank you very much,

**Comment by AlexKuhnle** [comment]
No, but it can be easily integrated: First, add the `trainable=True` argument [here](https://github.com/reinforceio/tensorforce/blob/master/tensorforce/core/networks/layer.py#L726), then add and forward it to Linear [here](https://github.com/reinforceio/tensorforce/blob/master/tensorforce/core/networks/layer.py#L554), and finally pass the argument to the variable initializations [here](https://github.com/reinforceio/tensorforce/blob/master/tensorforce/core/networks/layer.py#L676) and [here](https://github.com/reinforceio/tensorforce/blob/master/tensorforce/core/networks/layer.py#L692). PR welcome... :-)
